treeherder/model/sql/jobs.json

```diff
@@ -98,13 +98,13 @@
         },
         "set_result_set":{
 
-            "sql":"INSERT INTO `result_set` (`author`, `revision_hash`,`push_timestamp`, `active_status`)
-                SELECT ?,?,?,?
+            "sql":"INSERT INTO `result_set` (`author`, `revision_hash`,`long_revision`,`short_revision`,`push_timestamp`, `active_status`)
+                SELECT ?,?,?,?,?,?
                 FROM DUAL
                 WHERE NOT EXISTS (
                     SELECT `revision_hash`, `push_timestamp`
                     FROM `result_set`
-                    WHERE `revision_hash` = ?
+                    WHERE `revision_hash` = ? or `long_revision` = ? or `short_revision` = ?
```

Comment:
Hmm thought they are `OR`s, I read it as `AND`s.

I wonder if it's still best to leave the WHERE the same, for perf reasons for now?